app/services/katello/pulp/repository/yum.rb

```diff
@@ -60,9 +60,24 @@ def regenerate_applicability
           smart_proxy.pulp_api.extensions.repository.regenerate_applicability_by_ids([repo.pulp_id], true)
         end
 
+        def solve_dependencies_config(options, config = {})
```

Comment:
why does config need to be an extra parameter here. I dont see it being used anywhere